yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -589,36 +589,38 @@ def pixelize_cylinder(np.float64_t[:,:] buff,
                 r_i += 0.5*dx
             theta_i += dthetamin
 
-cdef int aitoff_thetaphi_to_xy(np.float64_t theta, np.float64_t phi,
+cdef int aitoff_Lambda_btheta_to_xy(np.float64_t Lambda, np.float64_t btheta,
                                np.float64_t *x, np.float64_t *y) except -1:
-    cdef np.float64_t z = math.sqrt(1 + math.cos(phi) * math.cos(theta / 2.0))
-    x[0] = math.cos(phi) * math.sin(theta / 2.0) / z
-    y[0] = math.sin(phi) / z
+    cdef np.float64_t z = math.sqrt(1 + math.cos(btheta) * math.cos(Lambda / 2.0))
+    x[0] = 2.0 * math.cos(btheta) * math.sin(Lambda / 2.0) / z
+    y[0] = math.sin(btheta) / z
     return 0
 
 @cython.cdivision(True)
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def pixelize_aitoff(np.float64_t[:] theta,
-                    np.float64_t[:] dtheta,
-                    np.float64_t[:] phi,
-                    np.float64_t[:] dphi,
+def pixelize_aitoff(np.float64_t[:] azimuth,
```

Comment:
I'm pretty ambivalent about the idea of changing all the notation in here, but I suppose it *does* make things clearer.